45c93c87bde453df1062e6fce498fb9d1708b73e
src/main/java/com/yelp/nrtsearch/server/grpc/LuceneServer.java
--- 
+++ 
@@ -19,10 +19,12 @@
 
 package com.yelp.nrtsearch.server.grpc;
 
+import com.google.api.HttpBody;
 import com.google.inject.Guice;
 import com.google.inject.Inject;
 import com.google.inject.Injector;
 import com.google.protobuf.ByteString;
+import com.google.protobuf.Empty;
 import com.yelp.nrtsearch.LuceneServerModule;
 import com.yelp.nrtsearch.server.MetricsRequestHandler;
 import com.yelp.nrtsearch.server.config.LuceneServerConfiguration;
@@ -763,10 +765,10 @@
         }
 
         @Override
-        public void metrics(MetricsRequest request, StreamObserver<MetricsResponse> responseObserver) {
-            try {
-                MetricsResponse reply = new MetricsRequestHandler(collectorRegistry).process();
-                logger.debug("MetricsResponse returned " + reply.toString());
+        public void metrics(Empty request, StreamObserver<HttpBody> responseObserver) {
+            try {
+                HttpBody reply = new MetricsRequestHandler(collectorRegistry).process();
+                logger.debug("MetricsRequestHandler returned " + reply.toString());
                 responseObserver.onNext(reply);
                 responseObserver.onCompleted();
             } catch (Exception e) {